Create launching.md

TomStone95 opened the pull request:
So this is kind of specific to getting Jarvis to function in Slack. In the future, we could maybe split this up so that the launching.md is generic, and then include separate short guides on how to integrate with slack, salesforce, etc.

TomStone95 committed:
Create launching.md

So this is kind of specific to getting Jarvis to function in Slack. In the future, we could maybe split this up so that the launching.md is generic, and then include separate short guides on how to integrate with slack, salesforce, etc.

tiagosiebler commented on _notes/launching.md:
```diff
@@ -0,0 +1,50 @@
+##Launching Jarvis in Slack
+
+###Create a Testing HTTP Server With ngrok.io 
```
For headers, put a space after the hashtags before the title starts. Otherwise the markdown syntax doesn't recognise it. E.g:
```
### Create a Testing HTTP Server With ngrok.io 
```

tiagosiebler commented on _notes/launching.md:
```diff
@@ -0,0 +1,50 @@
+
+Common errors are 
+- not having the client id defined in the .env file (this will throw errors)
+- not having the OAuth Redirect url defined in the app.
```
do you remember what happens when this isn't done? what's the symptom(s)?

TomStone95 replied:
Don't remember the behavior, but it's very in-your-face. the web page tells you something was wrong.

tiagosiebler reviewed (changes requested):
some minor changes, good work!

TomStone95 committed:
Update launching.md

Changed the Headers. Under "Final App Setup" I can't remember what error messages came up when those two things were wrong.
